production/helm/loki/values.yaml

```diff
@@ -480,7 +480,7 @@ enterprise:
     admin_client:
       storage:
         s3:
-          bucket_name: admin
+          bucket_name: {{ .Values.loki.storage.bucketNames.admin }}
```

Comment:
good catch, looks like we broken this in #12548. Would you mind adding a default for this value _only_ in the case where both minio and enterprise are enabled? Otherwise there should be no default so we don't run into the problem fixed in #12548

Reply:
I've updated this now. I've separated the admin storage config out into its own separate Helm value (`enterprise.adminClientConfig`), which includes a default for Minio only. 

To set any other storage options (e.g. AWS, GCS, etc), users can just override that `adminClientConfig` value.